core/templates/dev/head/pages/exploration_editor/statistics_tab/multiple_incorrect_issue_directive.html

```diff
@@ -0,0 +1,34 @@
+<div class="row oppia-issues-row-eq-height oppia-issues-bor-left oppia-issues-issue-dir">
+  <div class="col-md-3 oppia-issues-bor-btm oppia-issues-bor-right text-center oppia-pt2 oppia-pb2 oppia-issues-bg-clr oppia-issues-text-white">
+    <div style="font-size: 1.3em;">
+      Issue <[currentIssueIdentifier]>
+    </div>
+  </div>
+  <div class="col-md-9 oppia-issues-bor-left text-center d-flex justify-content-center oppia-issues-bor-btm oppia-pt2 oppia-pb2">
+    <[issueStatement]>
```

Comment:
@brianrodri This requires an IssueSService change, to use $sce.trustAsHtml() and so it would be better to do this in a separate PR for all three directives

Reply:
SGTM.